source/docs/software/vision-processing/introduction/game-piece-detection.rst

```diff
@@ -0,0 +1,227 @@
+- **pitch**: The calculated pitch angle to the target
+
+.. important:: The camera mounting angle is critical for accurate distance measurements. A camera pointed straight ahead cannot accurately determine distance to objects at the same height. Angling the camera down 15-30 degrees typically provides good results for ground-level game pieces.
+
+### Estimating Target Pose
+
+For more advanced applications, you may want to estimate the full 3D pose (position and orientation) of the game piece relative to your robot. This requires:
+
+1. **Known dimensions** of the game piece
+2. **Calibrated camera** to account for lens distortion
+3. **Multiple features** on the object to determine orientation
+
+For circular game pieces, orientation may not matter. For asymmetric objects, detecting specific features (corners, markings, etc.) can help determine how the object is oriented.
+
+The WPILib pose estimation classes can incorporate vision measurements to help track your robot's position and plan paths to game pieces.
+
+### Alternative: Area-to-Distance Lookup
+
+If your game piece is always oriented the same way, you can create a lookup table correlating particle area (in pixels) to distance. This is less accurate than trigonometric methods but can be simpler to implement:
+
+1. Place the game piece at known distances from your camera
+2. Measure the particle area at each distance
+3. Create a lookup table or fit a curve to the data
+
+Note that area varies with the inverse square of distance, so small measurement errors at large distances can cause significant inaccuracy.
+
+## Choosing a Camera
+
+Selecting the right camera is crucial for reliable game piece detection. Key considerations include:
+
+### Field of View (FOV)
+
+- **Wide FOV (>100°)**: Better for detecting game pieces at close range and wide areas, but objects appear smaller at distance
+- **Narrow FOV (60-90°)**: Better for detecting distant objects with more detail, but limited peripheral vision
+- Consider your robot's typical operating distance and whether you need to see multiple game pieces simultaneously
+
+### Resolution
+
+- **Higher resolution (1920x1080 or more)**: Provides more detail for distant detection and better pose estimation, but requires more processing power
+- **Lower resolution (640x480 to 1280x720)**: Faster processing, sufficient for many traditional vision tasks
+- Balance resolution with frame rate - a 720p camera at 60 FPS may outperform a 1080p camera at 15 FPS
+
+### Frame Rate
+
+- **Higher frame rate (60+ FPS)**: Reduces motion blur, crucial for detecting pieces while moving
+- **Lower frame rate (15-30 FPS)**: Sufficient for stationary detection, uses less bandwidth
+
+### Exposure Control
+
+- **Manual exposure control**: Essential for consistent detection under varying lighting
+- **Global shutter vs rolling shutter**: Global shutter is preferable for fast-moving robots to avoid image distortion
```

Comment:
maybe something that says rolling shutter cameras are much more common and if it doesn't say it's global shutter, then it's rolling shutter